File: 941-sort-array-by-parity/solution.py
class Solution:
    def sortArrayByParity(self, A):
        """
        :type A: List[int]
        :rtype: List[int]
        """
        res = []
        for v in A:
            if v % 2 == 0:
                res.append(v)
        for v in A:
            if v % 2 != 0:
                res.append(v)
        return res;

File: 941-sort-array-by-parity/solution.cpp
class Solution {
public:
    vector<int> sortArrayByParity(vector<int>& A) {
        vector<int> res;
        for (auto v : A) {
            if (v % 2 == 0) res.insert(res.begin(), v);
            else res.emplace_back(v);
        }
        return res;
    }
};